zeppelin-interpreter-api/pom.xml

```diff
@@ -68,8 +68,6 @@
               <exclude>org.apache.commons:commons-exec</exclude>
               <!-- Leave log4j unshaded so downstream users can configure logging. -->
               <exclude>log4j:log4j</exclude>
-              <exclude>com.esotericsoftware:kryo</exclude>
```

Comment:
@zjffdu , I see that you excluded `kryo` in https://github.com/apache/zeppelin/commit/ae7318f4c443a1e0fb1e6bc4758ad99092b880c4.
But the cluster needs `kryo`, So I shaded and rename `kryo` in interpreter-api pom.
Please help me review here, See if it will affect the operation of spark, Thanks you!

Reply:
Let's see if this affects the test ZeppelinSparkClusterTest.java

Reply:
ok, [CI Pass](https://travis-ci.org/liuxunorg/zeppelin/builds/537437337), ZeppelinSparkClusterTest.java no problem.